Unable to complete setup, fails during open-pdks installation

@vvbandeira 

Using command `./scripts/setup-ci.sh 2022.03.15_02.27.26`
Installation failed with following message:
```
Cloning into '/home/vijayan/MPW_CI/OpenLane-MPW-CI/OpenLane/pdks/open_pdks'...
remote: Enumerating objects: 6398, done.
remote: Counting objects: 100% (2563/2563), done.
remote: Compressing objects: 100% (1121/1121), done.
remote: Total 6398 (delta 1790), reused 2028 (delta 1316), pack-reused 3835
Receiving objects: 100% (6398/6398), 14.09 MiB | 19.52 MiB/s, done.
Resolving deltas: 100% (4198/4198), done.
Already on 'master'
Unable to find image 'efabless/openlane:2b3ecccb5600a3391a0ede3e0871d65990ac1f7c' locally
docker: Error response from daemon: manifest for efabless/openlane:2b3ecccb5600a3391a0ede3e0871d65990ac1f7c not found: manifest unknown: manifest unknown.
See 'docker run --help'.
make: *** [build-pdk] Error 125
```
Attaching terminal log below.
[mpw_ci_setup.log](https://github.com/The-OpenROAD-Project/OpenLane-MPW-CI/files/8397309/mpw_ci_setup.log)

--- a/scripts/setup-ci.sh
+++ b/scripts/setup-ci.sh
@@ -14,5 +14,5 @@ curl -L "https://github.com/The-OpenROAD-Project/OpenLane/archive/refs/tags/${1}
   | tar --strip 1 -xzC OpenLane
 
 echo "[INFO] Starting PDKs setup..."
-make -C OpenLane pdk-with-sram -j 1 NPROC=1
+make OPENLANE_DOCKER_TAG=$1 -C OpenLane pdk-with-sram -j 1 NPROC=1
 echo "[INFO] PDKs setup complete."

